main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -86,12 +86,12 @@ object FlixPackageManager {
                   case _: IOException => return Err(PackageError.DownloadError(s"Error occurred while downloading $assetName"))
                 }
                 out.println(s"Installation of $assetName completed")
-                newDownloads.addOne(project + "/" + s"ver${release.version.toString}" + "/" + assetName)
               } else {
                 out.println(s"$assetName already exists")
               }
             }
-            Ok(newDownloads.toList)
+
+            Ok(installed)
```

Comment:
What would be more sensible to return than `List[String]`? Maybe `List[Path]` or some `List[Dependency]`? Thoughts?

Reply:
I guess that depends on what we need it for?

Reply:
OK right. Maybe we return `Unit` instead for now, and then revise if we need it.

Reply:
And then the same for the FlixPackageManager?

Reply:
Yeah